x64-windows-static:
~~~
FAILED: utilities/tcpstat/dv-tcpstat.exe 
cmd.exe /C "cd . && D:\downloads\tools\cmake-3.25.1-windows\cmake-3.25.1-windows-i386\bin\cmake.exe -E vs_link_exe --intdir=utilities\tcpstat\CMakeFiles\dv-tcpstat.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1434~1.319\bin\Hostx64\x64\link.exe  @CMakeFiles\dv-tcpstat.rsp  /out:utilities\tcpstat\dv-tcpstat.exe /implib:utilities\tcpstat\dv-tcpstat.lib /pdb:utilities\tcpstat\dv-tcpstat.pdb /version:0.0 /machine:x64 /nologo /DEBUG /INCREMENTAL:NO /OPT:REF /OPT:ICF   /subsystem:console  -DYNAMICBASE -NXCOMPAT -LARGEADDRESSAWARE -OPT:REF -OPT:ICF -GUARD:CF  && cd ."
LINK: command "C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1434~1.319\bin\Hostx64\x64\link.exe @CMakeFiles\dv-tcpstat.rsp /out:utilities\tcpstat\dv-tcpstat.exe /implib:utilities\tcpstat\dv-tcpstat.lib /pdb:utilities\tcpstat\dv-tcpstat.pdb /version:0.0 /machine:x64 /nologo /DEBUG /INCREMENTAL:NO /OPT:REF /OPT:ICF /subsystem:console -DYNAMICBASE -NXCOMPAT -LARGEADDRESSAWARE -OPT:REF -OPT:ICF -GUARD:CF /MANIFEST /MANIFESTFILE:utilities\tcpstat\dv-tcpstat.exe.manifest" failed (exit code 1319) with the following output:
qwindows.lib(qwindowscontext.cpp.obj) : error LNK2038: mismatch detected for '_COROUTINE_ABI': value '1' doesn't match value '2' in tcpstat.cpp.obj
qwindows.lib(qwindowstheme.cpp.obj) : error LNK2038: mismatch detected for '_COROUTINE_ABI': value '1' doesn't match value '2' in tcpstat.cpp.obj
Qt6Core.lib(qfactorycacheregistration.cpp.obj) : error LNK2038: mismatch detected for '_COROUTINE_ABI': value '1' doesn't match value '2' in tcpstat.cpp.obj
Qt6Core.lib(qlocale_win.cpp.obj) : error LNK2038: mismatch detected for '_COROUTINE_ABI': value '1' doesn't match value '2' in tcpstat.cpp.obj


   Creating library utilities\tcpstat\dv-tcpstat.lib and object utilities\tcpstat\dv-tcpstat.exp


utilities\tcpstat\dv-tcpstat.exe : fatal error LNK1319: 4 mismatches detected
~~~

@Neumann-A Do you have an idea if this can be resolved? This port uses `-std:c++20`. What does Qt6 use? (Pulled in here via opencv4.)

--- a/ports/dv-processing/cmake-project-include.cmake
+++ b/ports/dv-processing/cmake-project-include.cmake
@@ -1,0 +1,5 @@
+if(MSVC)
+    # This port's tools use C++20, but Qt6 (via OpenCV4) uses C++17.
+    # Assuming that no coroutines are passed between the two.
+    add_definitions(-D_ALLOW_COROUTINE_ABI_MISMATCH)
+endif()

--- a/ports/dv-processing/portfile.cmake
+++ b/ports/dv-processing/portfile.cmake
@@ -27,7 +27,7 @@ vcpkg_check_features(
 set(VCPKG_BUILD_TYPE release) # no lib binaries
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
-    DISABLE_PARALLEL_CONFIGURE # writes to include/cmake/version.hpp
+    DISABLE_PARALLEL_CONFIGURE # writes to include/dv-processing/version.hpp
     OPTIONS
         ${FEATURE_OPTIONS}
         "-DCMAKE_PROJECT_INCLUDE=${CMAKE_CURRENT_LIST_DIR}/cmake-project-include.cmake"

--- a/ports/dv-processing/portfile.cmake
+++ b/ports/dv-processing/portfile.cmake
@@ -30,6 +30,7 @@ vcpkg_cmake_configure(
     DISABLE_PARALLEL_CONFIGURE # writes to include/cmake/version.hpp
     OPTIONS
         ${FEATURE_OPTIONS}
+        "-DCMAKE_PROJECT_INCLUDE=${CMAKE_CURRENT_LIST_DIR}/cmake-project-include.cmake"
         -DCMAKE_DISABLE_FIND_PACKAGE_Git=ON
         -DCMAKE_REQUIRE_FIND_PACKAGE_lz4=ON
         -DCMAKE_REQUIRE_FIND_PACKAGE_zstd=ON
@@ -46,4 +47,5 @@ vcpkg_copy_tools(TOOL_NAMES dv-filestat dv-imu-bias-estimation dv-list-devices d
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/lib") # pkgconfig only, but incomplete
 
+file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
 vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")

--- a/ports/dv-processing/portfile.cmake
+++ b/ports/dv-processing/portfile.cmake
@@ -38,12 +38,12 @@ vcpkg_cmake_configure(
         -DENABLE_SAMPLES=OFF
         -DENABLE_PYTHON=OFF
         -DBUILD_CONFIG_VCPKG=ON
-#[[ WIP ]] -DENABLE_UTILITIES=ON
-#[[ WIP ]] -DVCPKG_TRACE_FIND_PACKAGE=ON
 )
 vcpkg_cmake_install()
 
-vcpkg_copy_tools(TOOL_NAMES dv-filestat dv-imu-bias-estimation dv-list-devices dv-tcpstat AUTO_CLEAN)
+if(ENABLE_UTILITIES)
+    vcpkg_copy_tools(TOOL_NAMES dv-filestat dv-imu-bias-estimation dv-list-devices dv-tcpstat AUTO_CLEAN)
+endif()
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/lib") # pkgconfig only, but incomplete
 

--- a/ports/dv-processing/usage
+++ b/ports/dv-processing/usage
@@ -1,0 +1,6 @@
+dv-processing provides CMake targets:
+
+    find_package(dv-processing CONFIG REQUIRED)
+    target_link_libraries(main PRIVATE dv::processing)
+
+Using dv-processing requires a C++20 compliant compiler.

--- a/ports/dv-processing/vcpkg.json
+++ b/ports/dv-processing/vcpkg.json
@@ -6,6 +6,7 @@
   "homepage": "https://gitlab.com/inivation/dv/dv-processing",
   "license": "Apache-2.0",
   "dependencies": [
+    "boost-algorithm",
     {
       "name": "boost-asio",
       "features": [
@@ -14,6 +15,8 @@
     },
     "boost-callable-traits",
     "boost-circular-buffer",
+    "boost-core",
+    "boost-endian",
     "boost-geometry",
     "boost-lockfree",
     "boost-nowide",

--- a/ports/dv-processing/vcpkg.json
+++ b/ports/dv-processing/vcpkg.json
@@ -44,7 +44,7 @@
   ],
   "features": {
     "tools": {
-      "description": "Build utilities"
+      "description": "Build CLI utilities"
     }
   }
 }

--- a/versions/d-/dv-processing.json
+++ b/versions/d-/dv-processing.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "eb0a76029b92d2df35bb050c66c6adf43ab6df54",
+      "git-tree": "d501b6b860a75321634d378924af617142341fce",
       "version": "1.7.8",
       "port-version": 1
     },

--- a/versions/d-/dv-processing.json
+++ b/versions/d-/dv-processing.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "d870e09729d36b26a63bd4cc38b0a04f5b6c1486",
+      "git-tree": "eb0a76029b92d2df35bb050c66c6adf43ab6df54",
       "version": "1.7.8",
       "port-version": 1
     },

